### PR TITLE
Document the use of scope with GHA caching

### LINKED ISSUE #706: Document scope attribute
Enhancement [per prior discussion](https://github.com/docker/build-push-action/issues/624#issuecomment-1141336628)

Still not documented in https://github.com/docker/build-push-action/blob/master/docs/advanced/cache.md#github-cache

### PR DESCRIPTION
**Pull request by yanirs**
Resolves #706.

### CODE REVIEW

File: docs/advanced/cache.md [hunk 1]
```diff
@@ -109,6 +109,9 @@ exclusively used in a GitHub Action workflow as the `url` (`$ACTIONS_CACHE_URL`)
 and `token` (`$ACTIONS_RUNTIME_TOKEN`) attributes are populated when a workflow
 is started.
 
+When building multiple images, add `scope=<scope_name>` to `cache-from` and `cache-to` to avoid
+overwriting the cache index. See https://github.com/moby/buildkit/issues/2885.
```
Comment (crazy-max) [inline docs/advanced/cache.md:113]: Better to link to https://docs.docker.com/build/building/cache/backends/gha/#scope

Comment (crazy-max) [inline docs/advanced/cache.md:113]: Better to link to https://docs.docker.com/build/building/cache/backends/gha/#scope

Comment (crazy-max) [inline docs/advanced/cache.md:113]: Better to link to https://docs.docker.com/build/building/cache/backends/gha/#scope

**Review by crazy-max — CHANGES_REQUESTED** [review]

File: docs/advanced/cache.md (hunk 1 above)
Comment (yanirs) [reply docs/advanced/cache.md]: Good point. Updated.

**Review by crazy-max — CHANGES_REQUESTED** [review]
Can you squash your commits please?

**Comment by yanirs** [comment]
Yeah, I actually do mind. This is a trivial change that'd help people who hit the same issue, but there was no need for it to have multiple review rounds or for it to be sitting as an open PR for a month. Specifically, you could have squashed the commits on your side or merged it after I squashed them.

I'm done here. If you want to move the extra paragraph to the other repo (which will still help people), you can do it.

**Pull request closed without merging**